docs/backend/guides/open-telemetry.md

```diff
@@ -0,0 +1,264 @@
+
+Similar to the otelmux library the [otelhttp library](https://github.com/trussworks/otelhttp) is designed to wrap HTTP handler functions. It has built in events (read & write) to report the request size, response size, and duration of the http request.  It is more configurable than the otelmux middleware allowing you to filter out certain routes and other controls.
+
+```golang
+otelhttp.NewHandler(router, "server-tls", []otelHTTP.Option{otelHTTP.ReadEvents, otelHTTP.WriteEvents})
+```
+
+Note: This implementation has been forked to the Trussworks GitHub to address bug fixes.
+
+#### SQL 
+
+The [XSAM/otelsql](https://github.com/XSAM/otelsql) library wraps the queries to our Postgres database and appends a span to the trace with the statement and duration information.  Query parameters may be hidden to not log sensitive information.  It can also be used to log calls to cursor rows when paging through larger result sets.
+
+#### Go
+
+The [Go runtime instrumentation library](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/runtime) primarily provides statistics on memory usage and garbage collection cycles that may degrade performance.
+
+## Data collection
+
+Data from the Milmove app is sent to the Open Telemetry collector, which processes the trace, metric, and log data and exports it to the services of our choice.  In AWS the collector exports to the Cloudwatch and AWS X-Ray services for storage and analysis.  When run locally the data is exported to the Elastic APM Server running Elasticsearch and Kibana. 
+
+### Traces
+
+*Common Trace fields*
+- ID
+- Parent ID
+- Trace ID
+- Name
+- Kind
+- Start time
+- End time
+- Status code
+- Status message
+
+#### HTTP Requests
+
+```
+http.client_ip
+http.flavor
+http.host
+http.method
+http.scheme
+http.server_name
+http.status_code
+http.target
+http.user_agent
+http.wrote_bytes
+net.host_ip
+net.host_port
+net.peer_ip
+net.peer_port
+net.transport
+```
+
+#### Database SQL Statements
+
+Spans/segments will be added to the current trace when database queries (Select) or executions (Update/Insert) are made within a request.
+
+**sql.conn.exec** (Update/Insert statements)
+- db.statement
+
+**sql.conn.query** (Select statements)
+- db.system
+- db.statement
+
+**sql.rows** (When using a rows cursor to iterate through database results)
+- db.system
+
+### Metrics
+
+#### HTTP
+HTTP Request Count
+- HTTP Protocol Version
+- Host
+- Scheme
+- Server Name (TLS/MTLS/No-TLS)
+- Status Code
+- Target (URL path)
+
+HTTP Request Content Length
+- HTTP Protocol Version
+- Host
+- Scheme
+- Server Name (TLS/MTLS/No-TLS)
+- Target (URL path)
+
+HTTP Response Content Length
+- HTTP Protocol Version
+- Host
+- Scheme
+- Server Name (TLS/MTLS/No-TLS)
+- Target (URL path)
+
+HTTP Duration
+- HTTP Protocol Version
+- Host
+- Scheme
+- Server Name (TLS/MTLS/No-TLS)
+- Target (URL path)
+
+#### Database Connections
+
+Database Pool Connections
+- Inuse
+- Idle
+- Wait Duration
```

Comment:
Along the lines of mentioning the others are semantic conventions, idk if there's value in also mentioning these three are custom ones for us.